cardano-node/src/Cardano/Tracing/ToObjectOrphans.hs

```diff
@@ -842,6 +851,34 @@ instance ToObject (TraceLocalTxSubmissionServerEvent blk) where
   toObject _verb _ =
     mkObject [ "kind" .= String "TraceLocalTxSubmissionServerEvent" ]
 
+instance (Show (GenTx blk), Show (GenTxId blk), Show (ApplyTxErr blk))
+      => ToObject (TraceEventMempool blk) where
+  toObject _verb (TraceMempoolAddTxs txs mpSz) =
+    mkObject
+        [ "kind" .= String "TraceMempoolAddTxs"
+        , "txsAdded" .= String (pack $ show $ txs)
+        , "mempoolSize" .= String (pack $ show $ mpSz)
```

Comment:
Using `show` here kind of sucks, so I'm going to introduce some `ToObject` instances for the required types.